hack/print-resources

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+AWS_REGION="$1"
```

Comment:
love this, do you have a sample output?

Reply:
Yea sure. Had to change the implementation as I encountered issues with some resources. Let me know if you have any suggestions.
```
backup-076c90689bf3ca1d4
fs-067e2a8be6372f9e0
fsvol-0321424334e0b5eb9
backup-0df288c66bedb950c
fs-033ec9c52e120e117
fsvolsnap-0e2825f52f90d3249
fsvol-0fa0c73188f2f5290
```